### PR TITLE
Utilities: Port to LibMain

### PR DESCRIPTION
**Pull request by alexmajor**
Port shuf, shot, shutdown, sleep, sort, and sql to LibMain

### CODE REVIEW

File: Userland/Utilities/CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ list(APPEND SPECIAL_TARGETS test install)
 list(APPEND REQUIRED_TARGETS
     arp base64 basename cat chmod chown clear comm cp cut date dd df diff dirname dmesg du echo env expr false fgrep
     file find grep groups head host hostname id ifconfig kill killall ln logout ls mkdir mount mv nproc
-    pgrep pidof ping pmap ps readlink realpath reboot rm rmdir seq shutdown sleep sort stat stty su tail test
+    pgrep pidof ping pmap ps readlink realpath reboot rm rmdir seq shuf shutdown sleep sort stat stty su tail test
```
Comment (linusg) [inline Userland/Utilities/CMakeLists.txt:6]: ...what? That definitely belongs in its own commit.

File: Userland/Utilities/CMakeLists.txt [hunk 2]
```diff
@@ -136,10 +136,14 @@ target_link_libraries(pro LibMain LibProtocol)
...
-target_link_libraries(shot LibGUI)
-target_link_libraries(sql LibLine LibSQL LibIPC)
+target_link_libraries(shot LibGUI LibMain)
+target_link_libraries(shuf LibMain)
 target_link_libraries(stat LibMain)
 target_link_libraries(strace LibMain)
+target_link_libraries(shutdown LibMain)
+target_link_libraries(sleep LibMain)
+target_link_libraries(sort LibMain)
+target_link_libraries(sql LibLine LibMain  LibSQL LibIPC)
```
Comment (linusg) [inline Userland/Utilities/CMakeLists.txt:146]: Please keep this sorted

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Utilities/CMakeLists.txt (hunk 1 above)
Comment (alexmajor) [reply Userland/Utilities/CMakeLists.txt:6]: Removed.

File: Userland/Utilities/CMakeLists.txt (hunk 2 above)
Comment (alexmajor) [reply Userland/Utilities/CMakeLists.txt:146]: Sorted correctly.

**Review by kennethmyhra — CHANGES_REQUESTED** [review]
As pointed out in #12034, these should be split up into separate commits, one per application

**Comment by alexmajor** [comment]
Closed as commit are within #12034
